src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx

```diff
@@ -0,0 +1,35 @@
+export const SortbyRadio = ({ sortArray }) => {
+    return (
+        <>
+            <h3 className="heading-5">Sort By</h3>
+            <span>
+                {sortArray.map((sort) => (
+                    <label
+                        key={sort}
+                        htmlFor={sort}
+                        className="rootShoot-margin-center text-body-lg"
+                    >
+                        <input
+                            name="sort"
+                            id={sort}
+                            type="radio"
+                            value={sort}
+                            // checked={state.sortStrategy === sort}
+                            // onChange={(e) =>
+                            //     dispatch({
+                            //         type: "SORT_BY_PRICE",
+                            //         payload: { sortWay: e.target.value },
+                            //     })
+                            // }
+                        />
+                        {` Price - 
```

Comment:
This logic code can be removed from here. You can make a separate function for this functionality and call it here.